packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -157,6 +169,29 @@ export function WithdrawalConfirmationDialog(
                   onChange={setCheckbox2Checked}
                 />
 
+                {fastWithdrawalActive && (
+                  <Checkbox
+                    label={
+                      <span className="font-light">
+                        I understand that ~{confirmationPeriod} is an estimate,
```

Comment:
even though it's 30 mins, the dialog rounds it up to ~an hour

<img width="852" alt="image" src="https://github.com/user-attachments/assets/99b9a15f-96d4-438a-a420-b2ad9c80eeed">

Reply:
same on tx history